go.mod

```diff
@@ -122,6 +122,7 @@ require (
 	github.com/dop251/goja v0.0.0-20211011172007-d99e4b8cbf48 // indirect
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
 	github.com/elastic/gosigar v0.14.2 // indirect
+	github.com/ferranbt/fastssz v0.0.0-20210120143747-11b9eff30ea9 // indirect
```

Comment:
As you can see, we can't entirely get rid of `ferranbt/fastssz` because other libraries depend on it (I am 100% sure that `wealdtech` does, maybe some others too).

Reply:
The main reason for this PR is to prevent issues such as https://github.com/prysmaticlabs/prysm/issues/10153